mage_ai/frontend/components/FileBrowser/index.tsx

```diff
@@ -393,12 +394,13 @@ function FileBrowser({
     const {
       x: xContainer,
       width,
+      height,
```

Comment:
`height` here may not consistently get you the correct offset for calculating the context menu position. You might be able to use the `y` prop from the `coordinates` state to help with your calculation.

Reply:
This helped, you're awesome, mate. It was a bit misleading at first, because the ref is used at 2 different places, but maybe it's documented somewhere I didn't know haha.